CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
   json serializable of models and added copyWith method (Message, Reaction and Reply Message).
 * **Breaking**: [181](https://github.com/SimformSolutionsPvtLtd/flutter_chatview/pull/181) Removed
   deprecated field `showTypingIndicator` from ChatView.
+* **Breaking**: [187](https://github.com/SimformSolutionsPvtLtd/flutter_chatview/pull/187) update sendBy
```

Comment:
The PR number is 188. Can you please update this?

Reply:
updated this in next pr